http/src/client/mod.rs

```diff
@@ -1528,6 +1566,229 @@ impl Client {
         DeleteWebhookMessage::new(self, webhook_id, token, message_id)
     }
 
+    /// Respond to an interaction, by ID and token.
+    pub fn interaction_callback(
+        &self,
+        interaction_id: InteractionId,
+        interaction_token: impl Into<String>,
+        response: InteractionResponse,
+    ) -> InteractionCallback<'_> {
+        InteractionCallback::new(&self, interaction_id, interaction_token.into(), response)
+    }
+
+    /// Edit the original message, by its token.
+    ///
+    /// Functions the same as [`update_webhook_message`].
+    ///
+    /// [`update_webhook_message`]: Client::update_webhook_message
+    pub fn update_interaction_original(
+        &self,
+        interaction_token: impl Into<String>,
+    ) -> Result<UpdateWebhookMessage<'_>, InteractionError> {
+        UpdateWebhookMessage::new_interaction(self, self.application_id(), interaction_token)
```

Comment:
Can we use unique requests instead of modifying webhook request files? I know they function the same but they're different routes.

Reply:
After having a second look there was an error, here it is actually the same route, but there was an error in the implementation that broke it. I don't think it make sense to have two complete implementaitions where the only difference is if there is a ApplicationId or a WebhookId.

Reply:
At the moment we don't have any other http files that cross-pollenate like these do. I would prefer if it was kept that way.